app/pages/home.cjsx

```diff
@@ -79,7 +79,7 @@ module.exports = React.createClass
     document.documentElement.classList.remove 'on-home-page'
 
   lastFourProjects: ->
-    @props.user.get("project_preferences", page_size: 4, sort: '+updated_at')
+    @props.user.get("project_preferences", page_size: 4, sort: 'updated_at')
```

Comment:
Pretty sure the confusion here is the difference in the JSON API spec and the implementation.  (`+fieldname` is the explicit form of ascending, `fieldname` is the implicit ascending)